buster/Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN set -x \
 	&& apt-get install -y --no-install-recommends --no-install-suggests \
 		wget=1.20.1-1.1 \
 		ca-certificates=20190110 \
-		lib32z1 \
+		lib32z1=1:1.2.11.dfsg-1\
```

Comment:
There's a space missing between the version and the slash, this will error out in build ;P

Reply:
Sorry i fast edit :D, added space